frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -123,7 +123,6 @@ export const updateParcel = (parcel: IParcel) => async (dispatch: Function) => {
       parcel,
     );
     dispatch(success(actionTypes.UPDATE_PARCEL, status));
-    dispatch(fetchParcelDetail(data));
```

Comment:
There is no need to make another AJAX request as the returned result will have this.